operations/exchangerates.md

```diff
@@ -1,54 +1,59 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
 # Exchange rates
 
 ## Get all exchange rates
 
-Returns all available exchange rates among currencies of the [Enterprise](configuration.md#enterprise).
+Returns all available exchange rates among currencies of the [Enterprise](https://mews-systems.gitbook.io/connector-api/operations/configuration/#enterprise).
 
 ### Request
 
 `[PlatformAddress]/api/connector/v1/exchangeRates/getAll`
 
 ```javascript
 {
-    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
-    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
-    "Client": "Sample Client 1.0.0"
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `ClientToken` | string | required | Token identifying the client application. |
-| `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `Ids` | array of string | optional |  |
```

Comment:
Do we need to document this? Probably yes, since every field should have some documentation, right?

Reply:
I have opened a PR to the backend to hide and deprecated this field. I've checked the logs and no one used it in the past month.

Reply:
I'd keep it, just maybe add the documentation destription that it's unique identifier of exchange rates?

Reply:
I will follow up on this in separate set of PRs.